doc/user/content/get-started/quickstart.md

```diff
@@ -10,7 +10,6 @@ aliases:
   - /katacoda/
   - /quickstarts/
   - /install/
-  - /get-started/
```

Comment:
Oh ... I did make this change because since we have get-started/_index.md (What is Materialize? page).  Not sure how this is working currently (since it obviously is, but, after splitting the pages, this alias made What is Materialize redirect to the quickstart page when I run this locally.